src/fpga_mezz.c

```diff
@@ -0,0 +1,702 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+/* mraa header */
+#include "mraa/i2c.h"
+
+#include "fpga_mezz.h"
+
+#define ARD_I2C_0_FUNC 0
+#define ARD_SPI_FUNC 1
+#define ARD_UART_FUNC 2
+#define ARD_I2C_1_FUNC 3
+
+#define I2C_BUS 0
+
+#define INPUT 1
+#define OUTPUT 0
+
+#define HIGH 1
+#define LOW 0
+
+#define FUNC_PASSTHROUGH 0
+#define FUNC_GPIO 1
+
```

Comment:
Add functions header comment for all functions.

Reply:
added